SIS_import/sis_set_url.rb

```diff
@@ -278,15 +307,15 @@ def ESB_APICall(url, authorization_string, content_type, request_type, param_has
 ## checks the result of ESB API call
 ## renew ESB token if necessary, and retry the failed call due to expired token
 ## return parsed JSON
-def parse_ESB_API_CALL_RESPONSE(response, url, content_type, request_type, param_hash)
+def parse_ESB_API_CALL_RESPONSE(response, url, content_type, request_type, param_hash,scope)
```

Comment:
Please add doc for the newly added "scope" variable.

Reply:
Done